Task8/src/api/video.js

```diff
@@ -0,0 +1,83 @@
+import { ActionType } from '../store/actionTypes';
+
+const API_KEY = 'AIzaSyALMiCr_df0KxwdiWxpGeQBSCrczG5RcRs';
+let nextPageToken;
```

Comment:
As I see, currently 'Load More' functionality doesn't work. (this variable isn't assigned). Please, save `nextPageToken` on Redux state.

Reply:
Fixed